packages/cli/src/commands/generate/function/templates/function.ts.template

```diff
@@ -0,0 +1,18 @@
+import type { APIGatewayEvent, Context } from 'aws-lambda'
+
+/**
+ * The handler function is your code that processes http request events.
+ * You can use return and throw to send a response or error, respectively.
+ * 
+ * @param { import('aws-lambda').APIGatewayEvent } event - an object which contains information from the invoker.
+ * @param { import('aws-lambda').Context } context - contains information about the invocation,
+ * function, and execution environment.
+ */
```

Comment:
```suggestion
 * @typedef { import('aws-lambda').APIGatewayEvent } APIGatewayEvent */
 * @typedef { import('aws-lambda').Context } Context */
 * @param { APIGatewayEvent } event - an object which contains information from the invoker.
 * @param { Context } context - contains information about the invocation,
 * function, and execution environment.
 */
```

Reply:
It doesn't seem like I can submit this.

Reply:
Sorry! Totally assumed you could. I will right now.